Architecture/authentification.md

```diff
@@ -0,0 +1,73 @@
+# Authentification
+Normes et standard pour l'authentification des différentes populations d'utilisateurs.
+
+## Préconisation principale pour l'authentification des populations
+|Population cible|Solution(s) préconisées|
+|---|---|
+|<b>Agent du MAS</b>|Solution maison|
```

Comment:
```suggestion
|**Agent du MAS**|Solution maison|
```

Je propose cette suggestion pour la consistence avec le reste du repo, et pour respecter la syntaxe du Markdown.

De manière générale, il faudrait toujours privilégier le Markdown au HTML, à moins que ça soit nécessaire (styling particulier, etc.).

Reply:
Complétement d'accord. Il faut juste faire attention car dans les cellules des tables on ne peut pas utiliser tout le format markdown mais seulement les liens, l'italique, le gras et les mots de code (i.e. avec `xxx`) (cf. https://www.markdownguide.org/extended-syntax/#formatting-text-in-tables) le reste il faut le faire avec des balises html.